doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -117,40 +116,42 @@ you will need to rebuild Chapel from source in a different configuration.
       # re-build Chapel
       make
 
-      # make check is available but optional
-      make check
-
       # compile a sample program
-      chpl -o hello examples/hello.chpl
+      chpl -examples/hello3-datapar.chpl
 
       # run the sample program
-      ./hello
+      ./hello3-datapar
 
-   Note that the environment settings from ``util/setchplenv.bash`` will not persist beyond this terminal session.
-   You can choose to source ``setchplenv.bash`` whenever you want to use relevant commands.
-   If you want these environment settings to persist for future terminal sessions,
-   copy the commands from the Recommended Settings in :ref:`chplenv.rst <readme-chplenv.recommended_settings>` into your ``~/.bashrc`` file.
-   You can also store Chapel configuration settings in a :ref:`chplconfig <readme-chplenv.chplconfig>` file instead of your ``~/.bashrc``.
+   If you run into any portability issues, please see
+   :ref:`readme-bugs`:
 
-   See :ref:`chplenv.rst <readme-chplenv>` for a complete description of
-   Chapel's configuration variables, what they mean, and how they
-   can be set.
+
+Using Chapel in a Different Shell / Session
+-------------------------------------------
+
+Note that in both the Quickstart and preferred modes above, any
+environment settings made by ``setchplenv.bash`` will not persist
+beyond your current shell / terminal session.  To use Chapel from a
+different shell or terminal, you will need to either re-``source`` the
+``setchplenv.bash`` script that you used when building Chapel or set
+up your environment to support additional shells automatically.  See
+:ref:`using-chapel-in-other-sessions` for details.
 
 
 Using Chapel in Multi-Locale Mode
 ---------------------------------
 
-All of the instructions above describe how to run Chapel programs
-in a single-locale (shared-memory) mode. To run using multiple
-locales (distributed memory), please refer to
-:ref:`multilocale.rst <readme-multilocale>`.
+All of the instructions above describe how to run Chapel programs in a
+single-locale (shared-memory) mode. To run using multiple locales
+(distributed memory), please refer to :ref:`readme-multilocale`.
```

Comment:
Hmmm, so running in single locale mode is the "preferred, full-feature mode"?  Maybe above when we say preferred mode, we could say the preferred mode on a single node?  It would be a bit annoying for an HPC programmer to get to this point and realize they need to rebuild Chapel yet again to get what they want.

Reply:
Good point.  I've inserted a bullet before the "Repeat steps 2–5 above" bullet that says:

> * If you are interested in building Chapel to support multiple locales
>   (distributed memory execution), refer to :ref:`readme-multilocale`
>   for other settings to	enable that.